src/main/java/io/supertokens/webserver/api/session/SessionAPI.java

```diff
@@ -143,6 +144,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
             super.sendJsonResponse(200, result, resp);
         } catch (AccessTokenPayloadError e) {
             throw new ServletException(new BadRequestException(e.getMessage()));
+        } catch (UnauthorisedException e) {
```

Comment:
catching Unauthorised and returning a different status could get confusing. Create a new exception type for this.

Reply:
sure, okay

Reply:
done